docs/int/quickstart/advanced/quickstart-builder-api.md

```diff
@@ -27,79 +27,47 @@ charon run --builder-api
-
 ### Verify Charon + Builder API is Functional
 
-Once you have Charon and your Validator Client set up you can verify the set up is functional by reviewing your proposed blocks on Etherscan testnets or on via the Relay API endpoints.
+Once you have Charon and your Validator Client set up, you can verify if your setup is functional by reviewing your proposed blocks on [beaconcha.in](https://beaconcha.in) dashboards or via the Relay API endpoints.
 
-As an example if my validator was the block proposer for block 12853375 on Ropsten I can review the following resources.
+As an example, if my validator was the block proposer for block `17370078` on mainnet, I can review the following resources:
 
-[Etherscan Ropsten Block 12853375](https://ropsten.etherscan.io/block/12853375), if we check the `Extra Data` field on this page we will see the tag `Flashbots flashblock (Hex:0x466c617368626f747320666c617368626c6f636b)`. Relays will generally add a tag to the block, this block was submitted via the Flashbots Relay and as a result it has the tag: `Flashbots flashblock`.
+* [Beaconcha.in](https://beaconcha.in):
+  * Consider this [Mainnet block 17370078](https://beaconcha.in/block/17370078).
+  * If we check the `Block Extra Data` field under `Execution Payload`, we will see the tag `Illuminate Dmocratize Dstribute` (Hex:`0x496c6c756d696e61746520446d6f63726174697a6520447374726962757465`). 
+  * Relays will generally add a tag to the block. Since this block was submitted via the Flashbots Relay, as a result it has the tag.
+  * All mainnet flashbots blocks have this tag `Illuminate Dmocratize Dstribute`.
+* [Relay API](https://flashbots.github.io/relay-specs/):
+  * If you navigate to the `Data API` section on the Relay API page, you will see an endpoint labeled `/relay/v1/data/bidtraces/proposer_payload_delivered`.
+  * You can add a query argument of `block_number` to this call to see if a block was submitted via that Relay.
+  * [Here](https://boost-relay.flashbots.net/relay/v1/data/bidtraces/proposer_payload_delivered?block_number=17370078) is the query for the example block 17370078.
+  * Blocks that have not been submitted to the Relay will return an empty array `[]`.
 
-[Relay Data API](https://flashbots.notion.site/Relay-API-Spec-5fb0819366954962bc02e81cb33840f5), if you navigate to the `Data API` section on the Relay Data API you will see an endpoint labeled `proposerPayloadsDelivered`. You are able to add a query argument of `block_number` to this call to see if a block was submitted via that Relay. [Here](https://builder-relay-ropsten.flashbots.net/relay/v1/data/bidtraces/proposer_payload_delivered?block_number=12853375) is the query for the example block 12853375. Blocks that have not been submitted to the Relay will return an empty array `[]`.
+:::caution
+Note that the mainnet block in the above description is taken only for representation, and not actually proposed by a distributed validator.
+:::caution
```

Comment:
```suggestion
:::
```

Nit, don't think you need `caution` in the closing tag: https://docusaurus.io/docs/markdown-features/admonitions